cloudops/logging/google/logging.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import colorlog
```

Comment:
The same way, we don't want to import this dependency in production code right? Only for the local case

Reply:
Moved the colorlog import to the local logger constructor and added a try except in case the dev dependencies were not installed